packages/upswyng-web/src/components/__tests__/Resource.test.tsx

```diff
@@ -1,17 +1,24 @@
 import React from "react";
 import { Resource } from "../Resource";
-import { foodResource } from "../../DataMocks";
-import { shallow } from "enzyme";
+import { mount } from "enzyme";
 import useResource from "../useResource";
 
 jest.mock("../Categories", () => ({
   categories: {},
 }));
-jest.mock("../../utils/searchParams");
-jest.mock("../../constants");
-jest.mock("../LoadingSpinner.tsx", () => "LoadingSpinner");
+jest.mock("../LoadingSpinner.tsx", () => () => "TEST-loading-spinner");
+jest.mock("../PageBanner", () => () => "PageBanner");
+jest.mock("../Map", () => () => "Map");
+jest.mock("../Schedule", () => "Schedule");
+jest.mock("react-router", () => ({
+  useHistory: () => undefined,
+}));
```

Comment:
Rather than wrap the component within a router, I updated `useHistory` to be a noop instead. That way:

- react router never comes into play (within the test)
- whenever the component calls `useHistory`, it's still a function

If we ever wanted to test rendering based on what was returned from `useHistory`, the we would update the noop to be a `jest.fn()`, e.g.
```
jest.mock("react-router", () => ({
  useHistory: jest.fn(),
}));
```